File: assets/samples/email.verification/password/ruby/email_api_pass.rb
require 'open-uri'

email = 'support@whoisxmlapi.com'
password = 'your email verification api password'
username = 'your email verification api username'

url = 'http://www.whoisxmlapi.com/whoisserver/EmailVerifyService?'\
    + 'emailAddress=' + email + '&username=' +username + '&password=' +password\
    + '&validateDNS=true&validateSMTP=true&checkFree=true&checkCatchAll=true'\
    + '&checkDisposable=true&outputFormat=json'

puts open(url).read

File: assets/samples/email.verification/password/nodejs/email_api_pass.js
var http = require('http');

var email = 'support@whoisxmlapi.com';
var password = 'your email verification api password';
var username = 'your email verification api username';

var url = 'https://www.whoisxmlapi.com/whoisserver/EmailVerifyService?' +
    'emailAddress=' + email +
    '&validateDNS=true' +
    '&validateSMTP=true' +
    '&checkCatchAll=true' +
    '&checkFree=true' +
    '&checkDisposable=true' +
    '&username=' + username +
    '&password=' + password +
    '&outputFormat=' + format;

http.get(url, function(response) {
    var str = '';
    response.on('data', function(chunk) { str += chunk; });
    response.on('end', function() { console.log(str); });
}).end();